<launch>
	<node pkg="joy" name="joy_node" type="joy_node" required="true">
		<param name="autorepeat_rate" value="3" />
	</node>
	<node pkg="turtlebot_gamepad_training_replay" name="logicool_training" type="logicool_training.py" required="true" output="screen" />
	<node pkg="turtlebot_gamepad_training_replay" name="logger" type="logger.py" required="true" output="screen" />
	<node pkg="turtlebot_gamepad_training_replay" name="ledcontrol" type="ledcontrol.py" required="true" output="screen" />
	<node pkg="turtlebot_gamepad_training_replay" name="replay" type="replay" required="true" output="screen" />
</launch>
